packages/gunshi/src/definition.ts

```
@@ -15,116 +15,51 @@
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function define(definition: any): any {
-  if ('extensions' in definition && definition.extensions) {
-    const { extensions, run, ...rest } = definition
-    return {
-      ...rest,
-      run,
-      _extensions: extensions
-    }
-  }
+export function define<A extends Args = Args, E extends Record<string, unknown> = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)